### PR TITLE
Add backward navigation acceptance test

### PR DESCRIPTION
**Pull request by ScenarioFactory**
- Add backward navigation acceptance test

### CODE REVIEW

File: test/acceptance/features/example-grant/backward-navigation.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Feature: Backward Navigation
+
+    Scenario: Explore all backward navigation options
+        # start
+        Given the user navigates to "eligibility-checker/example-grant/start"
+        When the user clicks on "Start now"
+
+        # country
+        Then the user should be at URL "country"
+        When the user goes back
```
Comment (alanplatt) [inline test/acceptance/features/example-grant/backward-navigation.feature]: is this the back button or the browser, or both?

File: test/acceptance/steps/when.js [hunk 2]
```diff
@@ -18,3 +18,7 @@ When(/^(?:the user continues|continues)$/, async () => {
 When(/^(?:the user confirms|confirms) and sends$/, async () => {
   await $(`//button[@id='btnConfirmSend']`).click();
 });
+
+When(/^(?:the user goes|goes) back$/, async () => {
+  await $("//a[@class='govuk-back-link']").click();
```
Comment (alanplatt) [inline test/acceptance/steps/when.js:23]: ah, so its the back link.  Could be clearer?

Comment (ScenarioFactory) [reply test/acceptance/steps/when.js:23]: It could be, I just don't like 'And clicks on the back button'

Comment (ScenarioFactory) [reply test/acceptance/steps/when.js:23]: Maybe 'navigates backward'?

Comment (ScenarioFactory) [reply test/acceptance/steps/when.js:23]: Updated

**Review by alanplatt — APPROVED** [review]

**Pull request merged**